src/foam/nanos/menu/VerticalMenu.js

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
Use this license in the future:

/**
 * @license
 * Copyright 2017 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */